Source/WTF/wtf/glib/SysprofAnnotator.h

```diff
@@ -61,7 +61,7 @@ class SysprofAnnotator final {
     {
         auto key = std::make_pair(pointer, static_cast<const void*>(name.data()));
 
-        Vector<char, 1024> buffer;
+        Vector<char> buffer(1024);
         va_list args;
         va_start(args, description);
         vsnprintf(buffer.data(), 1024, description, args);
```

Comment:
Use `buffer.size()` here. And then `WTFMove(buffer)` when creating the value pair below.